docs/DeveloperGuide.md

```diff
@@ -239,6 +239,33 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### View Command Feature
+
+The view mechanism is facilitated by ``DoConnek Pro``
+
+It allows the user to view the details of a specific patient or specialist.
+
+Given below is an example usage scenario and how the view mechanism behaves at each step.
+
+Step 1. The user launches the application. The List of person will be initialized with the initial
+state, and the `Current Selected Person` pointer pointing to the first person on the list.
+![ViewState1](images/ViewState1.png)
+
+
+Step 2. The user executes `view 2` command to view the 2nd person details in the `DoConnek Pro`.
+The `view` command update the `Current Selected Person` pointer with the corresponding index from the
+input.
+![ViewState2](images/ViewState2.png)
+
+
+Step 3. The UI now update the change of the selected person and display the current selected person
```

Comment:
The phrasing here is a bit awkward in my opinion, maybe it could be changed to something like `The UI now updates the view panel to display the newly selected person and their details.`